batch2/batch/driver/instance_pool.py

```diff
@@ -194,17 +237,34 @@ def token_machine_name(self, inst_token):
             },
         }
 
-        await self.driver.gservices.create_instance(config)
-        log.info(f'created machine {machine_name} with logs at {self.worker_logs_directory}/{inst_token}/worker.log')
+        await self.gservices.create_instance(config)
```

Comment:
Yes, the heal loop (coming) would notice it is missing and update the database.

I considered putting the create instance first.  Then, if the database update fails, the activation is ignored because the instance isn't recognized and the instance kills itself.  Instance health status + healing loop seems preferrable.